app/views/issues/_page.html.erb

```diff
@@ -0,0 +1,40 @@
+<turbo-frame id="pagination" data-turbo-action="advance">
```

Comment:
This shouldn't have `data-turbo-action` but should have `target="_top"` and the advance action (and `pagination` as a target) should be on the next/previous buttons (which #4646 takes care of) so that other links inside the pagination section will break out to a full page load.

Reply:
I agree with you. I wrote it this way for it to be both functional and not to conflict with https://github.com/openstreetmap/openstreetmap-website/pull/4646 after merging, but I'll change it back.